Lib/test/test_keyword.py

```diff
@@ -20,17 +20,34 @@ def test_changing_the_kwlist_does_not_affect_iskeyword(self):
         keyword.kwlist = ['its', 'all', 'eggs', 'beans', 'and', 'a', 'slice']
         self.assertFalse(keyword.iskeyword('eggs'))
 
+    def test_changing_the_softkwlist_does_not_affect_issoftkeyword(self):
```

Comment:
The existence of this test and the above seem to contradict the rationale for changing isList to isSequence below (which I agree with).  But changing the above would be a different issue and probably not worth while.

Reply:
I agree with you.
Should i revert usage of `isSequence`?
Create new issue with this changes really seems not worth.